app/services/api/v1/participant_outcomes_query.rb

```diff
@@ -3,24 +3,37 @@
 module Api
   module V1
     class ParticipantOutcomesQuery
-      attr_reader :cpd_lead_provider
+      attr_reader :cpd_lead_provider, :participant_external_id
 
-      def initialize(cpd_lead_provider:)
+      def initialize(cpd_lead_provider:, participant_external_id: nil)
         @cpd_lead_provider = cpd_lead_provider
+        @participant_external_id = participant_external_id
       end
 
       def scope
+        if participant_external_id.nil?
+          return ParticipantOutcome::NPQ
+            .joins(:participant_declaration)
+            .order(:created_at)
+            .merge(declarations_scope)
+        end
+
         ParticipantOutcome::NPQ
-          .joins(:participant_declaration)
+          .joins(participant_declaration: { participant_profile: :participant_identity })
           .order(:created_at)
           .merge(declarations_scope)
+          .merge(participant_scope)
```

Comment:
To avoid duplication can we try something like this?
```
scope = ParticipantOutcome::NPQ.joins ....
scope = scope.merge(participant_scope) if participant_external_id.nil?
scope
```